src/main/java/oracle/kubernetes/operator/rest/resource/UpgradeApplicationsResource.java

```diff
@@ -0,0 +1,66 @@
+// Copyright 2018, Oracle Corporation and/or its affiliates.  All rights reserved.
+// Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+package oracle.kubernetes.operator.rest.resource;
+
+import oracle.kubernetes.operator.logging.LoggingFacade;
+import oracle.kubernetes.operator.logging.LoggingFactory;
+import oracle.kubernetes.operator.rest.model.UpgradeApplicationModel;
+import oracle.kubernetes.operator.rest.model.UpgradeApplicationsModel;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.core.MediaType;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * UpgradeApplicationsResource is a jaxrs resource that implements the REST api for the
+ * /operator/{version}/domains/{domainUID}/upgradeApplications path.
+ * It can be used to upgrade a list of Java EE applications in a WebLogic domain.
+ */
+
+public class UpgradeApplicationsResource extends BaseResource {
```

Comment:
What is the return from this operation?  Is there a way to see status of the rolling?

Reply:
Good idea. Thinking about what to return and how ....